boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
> I don't understand this sentence. You are in control of boot.js (at this moment) but not of the code that is passing you a callback, expecting it to be called on an asynchronous way.

`$tw.loadTiddlersNode` is only called from `$tw.boot.startup` so I am in control of the code that is passing the callback because it is my own code in the same file. 

And yes, I am familiar with the event loop. I understand your concern, but writing the entire thing to be async will be quite a bit of work that I didn't really want to bring into core if we are not ready for it. But this change will allow me to write it to be async in my own software if I want to, without having to redistribute a modified version of tiddlywiki along with my own code.